### PR TITLE
Nabla.h import header

### PR DESCRIPTION
**Pull request by AnastaZIuk**

### CODE REVIEW

File: cmake/import/nabla.h.in [hunk 1]
```diff
@@ -1,160 +1,159 @@
...
+
+	//! The video namespace contains classes for accessing the graphics API
+	namespace video
+	{
+	}
+}
+
+/*! \file irrlicht.h
+	\brief Main header file of the irrlicht, the only file needed to include.
+*/
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:157]: remove

File: cmake/import/nabla.h.in [hunk 2]
```diff
@@ -1,160 +1,159 @@
...
+
+	//! All scene management can be found in this namespace: scene graph, scene nodes, cameras, animation, etc...
+	namespace scene
+	{
+	}
+
+	//! The video namespace contains classes for accessing the graphics API
+	namespace video
+	{
+	}
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:152]: we dont use doxygen, remove

File: cmake/import/nabla.h.in [hunk 3]
```diff
@@ -1,160 +1,159 @@
...
+
+	//! Creates an Irrlicht device with the option to specify advanced parameters.
+	/** Usually you should use createDevice() for creating an Irrlicht Engine device.
+	Use this function only if you wish to specify advanced parameters like a window
+	handle in which the device should be created.
+	\param parameters: Structure containing advanced parameters for the creation of the device.
+	See nbl::SIrrlichtCreationParameters for details.
+	\return Returns pointer to the created IrrlichtDevice or null if the
+	device could not be created. */
+	core::smart_refctd_ptr<IrrlichtDevice> createDeviceEx(const SIrrlichtCreationParameters& parameters);
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:123]: these dont exist anymore on the new API right @Crisspl ?

File: cmake/import/nabla.h.in [hunk 4]
```diff
@@ -1,160 +1,159 @@
...
+#include "vector2d.h"
+#include "vector3d.h"
+#include "vectorSIMD.h"
+#include "line3d.h"
+#include "matrix4SIMD.h"
+#include "position2d.h"
+#include "quaternion.h"
+#include "rect.h"
+#include "dimension2d.h"
+#include "EDriverTypes.h"
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:51]: header to kill

File: cmake/import/nabla.h.in [hunk 5]
```diff
@@ -1,160 +1,159 @@
...
+#include "ICameraSceneNode.h"
+#include "ICursorControl.h"
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:53]: is @sadiuk killing the cursor control @Crisspl ?

File: cmake/import/nabla.h.in [hunk 6]
```diff
@@ -1,160 +1,159 @@
...
+#include "IDummyTransformationSceneNode.h"
+#include "IEventReceiver.h"
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:55]: header to kill (and associated)

File: cmake/import/nabla.h.in [hunk 7]
```diff
@@ -1,160 +1,159 @@
...
+#include "splines.h"
+
+#include "SColor.h"
+#include "SCollisionEngine.h"
+#include "SExposedVideoData.h"
+#include "SIrrCreationParameters.h"
+#include "SKeyMap.h"
+#include "SViewFrustum.h"
+
+#include "SIrrCreationParameters.h"
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:81]: header to kill

File: cmake/import/nabla.h.in [hunk 8]
```diff
@@ -1,160 +1,159 @@
...
+#include "IVideoDriver.h"
+#include "IWriteFile.h"
+#include "Keycodes.h"
+#include "splines.h"
+
+#include "SColor.h"
+#include "SCollisionEngine.h"
+#include "SExposedVideoData.h"
+#include "SIrrCreationParameters.h"
+#include "SKeyMap.h"
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:78]: header kill

File: cmake/import/nabla.h.in [hunk 9]
```diff
@@ -1,160 +1,159 @@
...
+#include "path.h"
+#include "ISceneManager.h"
+#include "ISceneNode.h"
+#include "ISceneNodeAnimator.h"
+#include "ISceneNodeAnimatorCameraFPS.h"
+#include "ISceneNodeAnimatorCameraMaya.h"
+#include "ITimer.h"
+#include "IVideoDriver.h"
+#include "IWriteFile.h"
+#include "Keycodes.h"
```
Comment (devshgraphicsprogramming) [inline cmake/import/nabla.h.in:71]: header to kill

File: cmake/import/nabla.h.in (hunk 3 above)
Comment (Crisspl) [reply cmake/import/nabla.h.in:123]: yea

File: cmake/import/nabla.h.in (hunk 5 above)
Comment (Crisspl) [reply cmake/import/nabla.h.in:53]: yea, absolutely, it's never been used by us tbh

Comment (Crisspl) [reply cmake/import/nabla.h.in:53]: well, "killing", we're just not touching it

Comment (devshgraphicsprogramming) [reply cmake/import/nabla.h.in:53]: lets kill dead code, it makes me happy

**Pull request merged**